feeder/ethereum/src/LogClient.ts

```diff
@@ -0,0 +1,44 @@
+import got from "got";
+import { b64ToHex } from "./utils";
+
+// Encapsulates functionality related to interfacing with public log APIs.
+// Performs operations like getting current STHs or calculating consistency proofs.
+export default class LogClient {
```

Comment:
Similarly, how about `CTLogClient` (or `CTClient`), or having some "CT" indication in the directories structure?